hack/local-development/start-gardenlet

```diff
@@ -29,19 +31,90 @@ else
   trap cleanup_kubeconfig EXIT
 fi
 
+YQ="$REPO_ROOT/hack/tools/bin/yq"
+GARDEN_KUBECONFIG="${GARDEN_KUBECONFIG:-$kubeconfig}"
+
+# determine the seed to manage; if multiple seeds exist on the
+# garden, the SEED_NAME env variable must be set
+if [ -n "${SEED_NAME:-}" ]; then
```

Comment:
Proposal: 
Add a `SEED_NAME` var in `Makefile` that passes down to the env var if set, so that one can also run `make start-gardenlet SEED_NAME=my-seed`. This is a bit more obvious from the makefile already, shell completion can kick in if you've set it up, ...